prefect_aws/credentials.py

```diff
@@ -21,10 +19,20 @@ class AwsCredentials:
             This is only needed when you are using temporary credentials.
         profile_name: The profile to use when creating your session.
         region_name: The AWS Region where you want to create new connections.
+
+    Example:
+        Load stored AWS credentials:
+        ```python
+        from prefect_aws import AwsCredentials
+
+        aws_credentials_block = AwsCredentials.load("MY_BLOCK_NAME")
     """  # noqa E501
 
+    _block_type_name = "AWS Credentials"
+
```

Comment:
Also, I think need a `_logo_url` which Alex can help upload

Reply:
Yeah I couldn't find a good one, any thoughts?

Reply:
I usually get the one from the collections as a placeholder, and wait for Alex to upload for me.
https://github.com/PrefectHQ/orion/tree/main/docs/img/collections

So:
https://github.com/PrefectHQ/orion/blob/main/docs/img/collections/aws.png?raw=true

Reply:
Oh awesome, will do